.solcover.js

```diff
@@ -20,4 +18,9 @@ module.exports = {
 		grep: '@cov-skip', // Find everything with this tag
 		invert: true, // Run the grep's inverse set.
 	},
+	// Reduce instrumentation footprint - volume of solidity code
+	// passed to compiler causes it to crash (See discussion PR #732)
+	// Line and branch coverage will still be reported.
+	measureStatementCoverage: false,
+	measureFunctionCoverage: false,
```

Comment:
```suggestion
	measureFunctionCoverage: true,
```

Toggling this back on might be necessary. Looks like codecov is combining function and line coverage for the totals, and there are a lot of dropped "lines" in this report. 

The report for `Depot` from `develop` a few days ago looks like this:
![Screen Shot 2020-10-11 at 12 49 46 PM](https://user-images.githubusercontent.com/7332026/95688663-99639a00-0bc0-11eb-908a-02b657f41678.png)

But the report in this PR looks like:
![Screen Shot 2020-10-11 at 12 53 50 PM](https://user-images.githubusercontent.com/7332026/95688698-db8cdb80-0bc0-11eb-8a4e-a2817d64cc21.png)

Reply:
Or I think you could also keep both new options set to false and your future reports will just be relative to the new baseline.